### ISSUE TITLE
Add resizable textareas

### ISSUE THREAD

**Issue by asyncLiz** [material-components/material-web#1305]
**Is your feature request related to a problem? Please describe.**
`<mwc-textarea>` does not have a resize handle like native `<textarea>`

**Describe the solution you'd like**
Textareas should either be resizable by default, or static by default. Resizing should be toggle-able by a boolean property (e.g. `resize`)

**Describe alternatives you've considered**
Multiline (auto-growing) textareas are another variant that will be explored in #455.

**Additional context**
Originally MWC disabled resizable textareas since the `<textarea>` element could be resized outside of the bounds of the `<mwc-textarea>` element. With [recent additions in MDC](https://github.com/material-components/material-components-web/blob/27533c19e9c72c5a27a33aaa764c1b6a05175cf5/packages/mdc-textfield/_mixins.scss#L1378-L1390) this is no longer an issue, and resizing can be safely turned on.
